src/sentry/digests/backends/redis.py

```diff
@@ -245,6 +268,8 @@ def add(self, key, record):
             if should_truncate:
                 logger.info('Removed %s extra records from %s.', results[-1], key)
 
+            return results[-2 if should_truncate else -1]
```

Comment:
I don't like this at all, but I also really don't want to go through the work of wrapping the pipeline in something that responds with futures for each operation right now either.